tools/CMakeLists.txt

```diff
@@ -185,6 +185,11 @@ else()
 
   PCL_ADD_EXECUTABLE(pcl_tiff2pcd COMPONENT ${SUBSYS_NAME} SOURCES tiff2pcd.cpp)
   target_link_libraries(pcl_tiff2pcd pcl_common pcl_io)
+  if(${VTK_VERSION} VERSION_GREATER_EQUAL 9.0)
+    target_link_libraries(pcl_tiff2pcd VTK::Rendering${VTK_RENDERING_BACKEND} VTK::RenderingCore VTK::RenderingFreeType VTK::InteractionStyle)
+  else()
+    target_link_libraries(pcl_tiff2pcd vtkRendering${VTK_RENDERING_BACKEND} vtkRenderingCore vtkRenderingFreeType vtkInteractionStyle)
+  endif()
```

Comment:
Same as above: Why it compiled before this MR?

Reply:
All VTK was linked into IO - so same as above, after limiting to only link what is required to build IO, now shows explicitly what pcl_tiff2pcd really depends on. But maybe it can be looked into, why it does so.